R/derive_extreme_event.R

```diff
@@ -197,11 +361,20 @@
 #'     PARAMCD = "ALK2",
 #'     PARAM = "ALKPH <= 2 times ULN"
 #'   )
-#' )
+#' ) %>%
+#' select(-STUDYID)
 #'
-#' # Derive confirmed best overall response (using event_joined())
-#' # CR - complete response, PR - partial response, SD - stable disease
-#' # NE - not evaluable, PD - progressive disease
+#' @caption A more complex example: Confirmed Best Overall Response
```

Comment:
Do you want to show how to derive CBOR or how to add events requiring confirmation or how `cond_first_upper`, `join_type = "after"` and `dataset_name`/`source_datasets` work?
Depending on this we may need to adjust the title.

Reply:
I've added the arguments in the title, as i think it's mainly the last two options